src/platform/packages/shared/shared-ux/avatar/solution/src/solution_avatar.tsx

```diff
@@ -56,16 +61,27 @@ export const KibanaSolutionAvatar = (props: KibanaSolutionAvatarProps) => {
     icon.iconType = `logo${props.name.replace(/\s+/g, '')}`;
   }
 
+  const { euiTheme } = useEuiTheme();
+  const styles = {
+    base: css(useEuiShadow('s')),
+    xxl: css`
+      ${useEuiShadow('m')};
+      line-height: 100px;
+      width: 100px;
+      height: 100px;
+      border-radius: 100px;
+      display: inline-block;
+      background: ${euiTheme.colors.backgroundBasePlain} url('assets/texture.svg') no-repeat;
```

Comment:
This is making an invalid URL of `http://localhost:5601/app/assets/texture.svg` :(